### PR TITLE
Update based on voxpupuli/modulesync_config

### PR DESCRIPTION
**Pull request by dhoppe**
Please merge.

### CODE REVIEW

**Comment by jyaworski** [comment]
Rubocop.

**Comment by dhoppe** [comment]
I am already working on it. ;)

**Comment by dhoppe** [comment]
@jyaworski Done

File: spec/classes/collectd_plugin_logfile_spec.rb [hunk 1]
```diff
@@ -24,8 +24,7 @@
   context ':ensure => present, specific params, collectd version 4.9' do
     let :facts do
       { osfamily: 'Redhat',
-        collectd_version: '4.9.0'
-      }
+        collectd_version: '4.9.0' }
```
Comment (bastelfreak) [inline spec/classes/collectd_plugin_logfile_spec.rb]: this is really wonky. can you migrate that to:
```ruby
{
  osfamily: 'Redhat',
  collectd_version: '4.9.0'
}
```

also applies for the other hashes.

**Comment by bastelfreak** [comment]
besides the hash annotation this looks fine.

**Pull request merged**